lib/mbta.js

```diff
@@ -1,12 +1,43 @@
 'use strict'
 
-// Code here.
+/*
+Create a function in lib/mbta.js to calculaste the number of stops between stations on the "MBTA".
+Write additional functions used by this function as needed.
+
+The function takes the line and stop that a rider is getting on at and the line and stop that a rider
+is getting off at and returns the total number of stops for the trip.
+There are 3 subway lines:
+
+All 3 subway lines intersect at Park Street, but there are no other intersection points. Some of this MBTA is
+fictionalized.
+*/
 
 const stopsBetweenStations = (startLine, startStation, endLine, endStation) => {
 
+  //stops.orange.indexOf('Back Bay')
+  let station2 = stops.endLine.indexOf(endStation)
+  let station1 = stops.startLine.indexOf(startStation)
+  return abs(station2 - station1)
 }
 
 module.exports = {
   stopsBetweenStations,
   stretch: false
 }
+
+let stops = {
+  red: ["South Station", "Park Street", "Kendall", "Central", "Harvard", "Porter",
+  "Davis", "Alewife"],
+  green: ["Government Center", "Park Street", "Boylston", "Arlington", "Copley",
+"Hynes", "Kenmore"],
+orange: ["North Station", "Haymarket", "Park Street", "State", "Downtown Crossing",
+"Chinatown", "Back Bay", "Forest Hills"]
+}
+
+
```

Comment:
You're on the right track, you just need to focus on building out the logic of how your function will evaluate how many stops have to be traveled. `Math.abs` and `indexOf` are valuable tools here